service/history/api/multioperation/api.go

```diff
@@ -79,6 +99,39 @@ func Invoke(
 		startReq:           startReq,
 	}
 
+	// TODO(stephanos): remove again
+	defer func() {
+		if retErr != nil {
```

Comment:
I would put it behind namespace scoped dynamic config.

Reply:
Done 👍